libekb.C

```diff
@@ -155,6 +156,49 @@ static void get_HWPErrorInfo(const fapi2::ReturnCode& rc, HWP_ErrorInfo& hwp_err
     }
 }
 
+/*
+ * @brief Used to callout hardware error details.
+ *
+ * The defined api is used collect defined error info in error xml based on
+ * given hwp return code (error code)
+ *
+ * @param ffdc used to pass buffer to fill FFDC data
+ * @param hwp_errinfo used to pass buffer to fill error details
+ *
+ * @return void
+ */
```

Comment:
instead of isSbe use generic , like guard required or not?

Reply:
ok